generator/nidaqmx_python_generator/metadata/doc/functions.md

```diff
@@ -0,0 +1,194 @@
+# ``DAQmx`` function metadata format
+
+## Example
+
+```python
+'CreateAIAccel4WireDCVoltageChan': {
+        'c_function_name' : 'CreateAIAccel4WireDCVoltageChan', 
+        'calling_convention':'StdCall',
+        'description': ' Creates channel(s) that use a Wheatstone bridge to measure force or load. Use this instance with sensors whose...',
+        'isFactory': False,
+        'python_class_name':'AIChannelCollection',
+        'handle_parameter':
+        {
+            "accessor": "self._handle", 
+            "ctypes_data_type": "lib_importer.task_handle", 
+            "cviName": "taskHandle"
```

Comment:
`isFactory` and `cviName` are still camelCase.

Reply:
changed them to snake case